### ISSUE TITLE
stop suddenly (23.2, Qt6+Telescope Control)

### ISSUE THREAD

**Issue by tajicoji** [Stellarium/stellarium#3308]
### Expected Behavior
stable operation

### Actual Behavior
It worked fine at first, but after a while it quit the moment I touched the window (probably terminated abnormally before that)

### Steps to reproduce
1. Clean install stellarium-23.2-a69d242-qt6-win64.exe
2. Configure remote control settings
3. After restarting, specify M42 from the APT object browser and click the SHOW button>>M42 will be displayed in Stellarium
4. Start telescope control of Stellarium (ASCOM TelescopeSimulator for .NET)
5. GOTO to the coordinates specified by APT>>Confirm that it is linked to Stellarium

Works fine at first. Keep this state for a while and try clicking the Stellarium Window.

### System
* Stellarium version: stellarium-23.2-a69d242-qt6-win64
* Operating system: Windows 11
* Graphics Card: Inter(R) UHD Graphics

### Log files
[log.txt](https://github.com/Stellarium/stellarium/files/11963979/log.txt)

**Comment by gzotti** [comment]
Most likely duplicate of #2821.

**Comment by nowani** [comment]
I’m having a similar issue with Stellarium and ASCOM Platform 6.5. The issue occurs when telescope control is enabled and Alt-Tab. The laptop is an HP Folio 840 G8 I7 16GB. Intel Iris Xe graphics.

This issue was occurring on Windows 11. I thought maybe that was the issue so I reformatted the laptop with Windows 10 (22H2) and all updates. Reinstalled ASCOM Platform and telescope drivers for Skywatcher EQ6-R Pro. Installed Stellarium (version listed in issue). Configured ASCOM, opened Stellarium, added telescope. Tested slewing (working). Alt tabbed out of Stellarium and it still crashes.

Note that the only software I installed was Stellarium and telescope control.

**Comment by nowani** [comment]
This is actually a very interesting discussion about alt-tabbing and how it can cause programs to crash. A lot of good technical details. An older version of Stellarium (.22.1) works on my laptop. In this newest version, something my have happened that’s not managing the change in state when Stellarium is Alt Tabbed out to a different application.

https://www.reddit.com/r/gamedev/comments/xtg6ug/why_are_some_games_more_prone_to_crash_when_i/#:~:text=if%20you%20are%20alt%2Dtabbing,is)%20to%20update%20itself%20appropriately.&text=threading%2C%20concurrency%2C%20async%20things%20can%20get%20jostled%20pretty%20hard.

**Comment by gzotti** [comment]
Interesting details. IIRC all reported incidents were with Qt6 based builds on Windows. Are there any other common properties that may relate to drivers: Always Intel GPU, always ASCOM, particular ASCOM devices? Was this with the classical or the new sky model? ...

**Comment by nowani** [comment]
It seems to be related to ASCOM and Stellarium running together with a connection. I found even the ASCOM simulator with a simulated connection crashes. Stellarium with its own simulator does not crash with Alt Tab switching. 

I’ve been out of the hobby for about 3 years and never had these kind of issues before with Stellarium. I decided to test with my old laptop (8 year old Dell Inspiron with I7 and 32 Gb ram). This is the old laptop I used to  control my telescope without issue. I updated the laptop today to current Stellarium (23.2) as well as the ASCOM Platform to 6.5. I can repeat the crash on this old laptop when Using an ASCOM connection with Stellarium. It crashed when I Alt Tab To another app.

**Comment by gzotti** [comment]
Does this have Intel, NVidia or AMD GPU?

**Comment by freekinfrog** [comment]
Not trying to butt in but ascom has updated to 6.6sp1..i was not sure it the OP made a mistake in stating his current build of 6.5 ascom

**Comment by alex-w** [comment]
Please check the fresh version (development snapshot) of Stellarium:
https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot

**Comment by gzotti** [comment]
Several Qt6/TelescopeControl issues have been solved before 23.3. We would like to clean-out a large group of related issue reports which are hard to reproduce without actual telescopes or external software. If no follow-up to this happens by November 1, 2023, we will close this as tentatively solved.

**Comment by kantn8r** [comment]
I'm a Version 1.2.4 (Qt5) users that is hesitant to try a newer Qt6 version.  Do you suggest I try the 23.2 (Qt5) version? I see you still maintain the note "Like in the 1.* series, it seems the TelescopeControl plugin at least on Windows works better in the Qt5-based releases", so I'm assuming that Qt6 still has issues.

**Comment by 10110111** [comment]
The suggestion is to check the latest snapshot of the Qt6 version. A new version is going to be released soon, and we need a confirmation that the issues we think solved are actually solved.

**Comment by kantn8r** [comment]
OK, you are looking for a guinea pig to test a snapshot. I tried that several times back with 1.1 and 1.2 and the Qt6 versions did not work for me. Just to confirm, you need users (like me) that have had problems with Qt6 to install "stellarium-23.2-75a6ab4-qt6-win64.exe" and see if the bugs have been fixed? Sorry for the extra follow up but I'm an end user, not a programmer and this stuff is way over my head.

**Comment by gzotti** [comment]
Yes, we have found two issues in the Qt6 build after 23.2 was out. These fixes are available in current betas. Some users have already confirmed this solved their problems, but we can use more people testing with their equipment (which we don't have, so we developers cannot check or predict with any certainty whether their equipment works).

**Comment by kantn8r** [comment]
OK Georg, I've installed [stellarium-23.2-75a6ab4-qt6-win64.exe]. It's cloudy the next few nights so it may take a while to test my equipment.

**Comment by kantn8r** [comment]
Georg, I tested [stellarium-23.2-75a6ab4-qt6-win64.exe] last night with the CPC 1100, a Windows 10 laptop, CPWI 2.5.2 Final, PHD2 and SharpCap 4.1 using a USB connection to the laptop. I did not have any issues the Telescope Control plugin. Go-to's, telescope navigation within SharpCap and plate solving all worked fine.

I still need to test my other mount, a Losmandy G11G, that I run with a Ryzen 5 Mini-PC and Windows Remote Desktop. I will provide an update once I test the G11G.
